KUDU_INTEGRATION.md

```diff
@@ -0,0 +1,57 @@
+# Kudu Integration using dbt-impala
+
+The `dbt-impala` adapter allows you to use [dbt](https://www.getdbt.com/) along with [Apache Kudu](https://kudu.apache.org) and [Cloudera Data Platform](https://cloudera.com)
+
+
+## Getting started
+
+- [Install dbt](https://docs.getdbt.com/docs/installation)
+- Read the [introduction](https://docs.getdbt.com/docs/introduction/) and [viewpoint](https://docs.getdbt.com/docs/about/viewpoint/)
+
+### Requirements
+
+- In a CDP public cloud deployment, Kudu is available as one of the many Cloudera Runtime services within the Real-time Data Mart template.
+- To use Kudu, you can create a Data Hub cluster by selecting Real-time Data Mart template template in the Management Console.
+- Follow this [article](https://blog.cloudera.com/integrating-cloudera-data-warehouse-with-kudu-clusters) on integrating the created Kudu service with Impala CDW.  
+
+
+For general instructions, please follow [Readme](README.md) guidelines.
+
+## Supported features
```

Comment:
can we add this table to Readme.md, as there is only one additional column.

Reply:
Sure, added this change.